src/quo/components/text_combinations/standard_title/style.cljs

```diff
@@ -2,12 +2,9 @@
   (:require [quo.foundations.colors :as colors]))
 
 (def container
-  {:flex            1
```

Comment:
Flex 1 caused the container to grow if the parent was a flex too. We don't want it to grow, just be space between.

Reply:
this component is used in many places so is likely to break some things - we should check other uses of it 👍